src/extensions/surveys/components/QuestionTypes.tsx

```diff
@@ -210,17 +210,16 @@ export function MultipleChoiceQuestion({
     questionIndex,
     appearance,
     onSubmit,
-    choices,
     closeSurveyPopup,
 }: {
     question: MultipleSurveyQuestion
-    choices: string[]
     questionIndex: number
     appearance: SurveyAppearance
     onSubmit: (choices: string | string[] | null) => void
     closeSurveyPopup: () => void
 }) {
     const textRef = useRef(null)
+    const choicesRef = getDisplayOrderChoices(question)
```

Comment:
Convention not to call anything Ref if it's not a React/Preact ref - which suggests interacting with the DOM somehow (https://preactjs.com/tutorial/05-refs/)

```suggestion
    const choices = getDisplayOrderChoices(question)
```

Reply:
fixed now.